### PR TITLE
Add Hexlet icon

### LINKED ISSUE #9295: Request: Add Hexlet icon
### Brand Name

Hexlet

### Website

https://hexlet.io/

### Popularity Metric

The Similarweb rank is 50,571. See https://www.similarweb.com/website/hexlet.io/

### Official Resources for Icon and Color

svg: https://cdn2.hexlet.io/assets/icon_en-b9ac6d8c1e37efc126ee2530530a8779f08de97300afa49bd49a1d9ecce59919.svg
color: `#ffffff`, from the website header

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by johanla0**
**Issue:** closes #9295 

**Similarweb rank:**
The Similarweb rank is 50,571. See https://www.similarweb.com/website/hexlet.io/

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
- Used suggested full logo svg: https://github.com/simple-icons/simple-icons/pull/9296#discussion_r1295886519
- Cut unnecessary strokes as they use only 'H' letter as favicon and everywhere
- Updated to monochrome palette with Inkscape
- Optimized with SVGO
- Picked color of  `#116ef5` from the original colored svg accent color (on top of the logo)

### Preview
hexlet

### CODE REVIEW

File: icons/hexlet.svg
Comment (adamrusted) [inline icons/hexlet.svg:1]: I'm seeing [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/3ed04b83-d824-44b7-bce2-7e7d3c028b38) when comparing the source SVG in blue against your PR in black. Could you take another look at this using the icon found in the header (see below for code!).

```xml
<svg xmlns="http://www.w3.org/2000/svg" role="img" version="1.1" id="Layer_1" x="0" y="0" viewBox="0 0 250 71" xml:space="preserve" width="100" height="20"><title>Hexlet</title><style>.ct0{fill:currentcolor}</style><path class="ct0" d="M77 27c-13 0-22 10-22 22 0 13 9 22 22 22 9 0 17-4 20-14h-9c-2 4-6 6-11 6-7 0-12-4-13-11h34c1-14-8-25-21-25zM65 45c1-6 5-10 12-10 6 0 11 4 12 10H65zM141 29h-10l-9 14-9-14h-11l14 20-15 22h11l10-15 10 15h10l-15-21zM149 12h9v59h-9zM188 27c-12 0-21 10-21 22 0 13 9 22 21 22 9 0 17-4 21-14h-9c-2 4-6 6-11 6-7 0-13-4-14-11h34c1-14-7-25-21-25zm-12 18c1-6 6-10 12-10 7 0 11 4 12 10h-24zM248 61l-8 2c-6 0-9-3-9-9V36h18v-8h-18V17l-8 2v9h-8v8h8v18c0 12 6 17 15 17 5 0 10-2 12-3l-2-7zM37 21v19H9V21l-8-3v53h8V48h28v23h8V18z"></path><path style="fill:#136ef6" d="M46 9 23 0 0 9l23 8z"></path></svg>
```

Comment (johanla0) [reply icons/hexlet.svg:1]: Sure, will use this svg to update!

**Comment by johanla0** [comment]
Updated icon, description and preview

**Review by adamrusted — APPROVED** [review]
It's perfect! Thanks @johanla0 🎉

**Pull request merged**